src/Servers/Kestrel/test/InMemory.FunctionalTests/InMemory.FunctionalTests.csproj

```diff
@@ -5,7 +5,7 @@
     <ServerGarbageCollection>true</ServerGarbageCollection>
     <TestGroupName>InMemory.FunctionalTests</TestGroupName>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <DefineConstants>$(DefineConstants);IS_FUNCTIONAL_TESTS</DefineConstants>
+    <DefineConstants>$(DefineConstants);IS_FUNCTIONAL_TESTS;INMEMORY</DefineConstants>
```

Comment:
I don't think we can get rid of this and everything that references `INMEMORY` since this csproj doesn't include anything from `..\FunctionalTests\**\*.cs"`. I think only the Sockets.FunctionalTests and Libuv.FunctioanlTests projects need this.

Reply:
Ok, so `src\Servers\Kestrel\test\FunctionalTests` is really Transport.FunctionalTests in practice?

Reply:
Basically.